guides/practices/pair-programming.md

```diff
@@ -57,19 +66,73 @@ gracefully.
 
 ## Remote pairing
 
-Remote pairing requires a certain amount of preparation. Before starting the pairing session,
-determine who will be the "host". This is the developer who will be sharing their screen.
-It is the host's responsibility to ensure the following before the guest comes online:
+Rmeote pairing is done using a cloud pairing station. We use Cloud9. A pairing instance
+must first be provisioned for the project. See [guide here](https://github.com/civilcode/cloud9-bootstrap).
+
+Both pair members access the Cloud9 instance using their Amazon IAM account. The pair use
+the tool they want for the audio (Zoom, slack, ...).
+
+## Git user
+
+For pulling and pushing to Github, we use our CivilCode pairing account (`civilcode-pairing`).
+The first time a repo is pulled through the `https` handle, git asks for the username and password.
+By default, the credentials are cached in OSX keychain.
+
+When committing code, the pair uses the [multiple co-authors](https://help.github.com/articles/creating-a-commit-with-multiple-authors/) feature of GitHub.
+The pairing station has a list of possible co-authors in the `.gitmessage` file.
```

Comment:
We should add them to the `dotfiles` project?

Reply:
Added issue to dotfiles project.